### PR TITLE
Reg 65/login issues

### PR DESCRIPTION
**Pull request by jag-123**
added logout method and redirect to /login page

### CODE REVIEW

File: src/reducers/LoginReducer.jsx [hunk 1]
```diff
@@ -31,6 +31,10 @@ const LoginReducer = (state = initialState, action) => {
         allCourses: action.courses,
         loggedIn: true,
       });
+    case 'LOGOUT_USER':
```
Comment (arianaolson419) [inline src/reducers/LoginReducer.jsx:36]: This is going to have to be moved in the state, but it's small :)

File: src/actions/actions.jsx [hunk 2]
```diff
@@ -112,6 +118,12 @@ export const getCourses = data => (
   }
 );
 
+// logout
+export const logout = data => (
+  (dispatch) => {
+    $.get('/logout', data)
+      .done(response => (dispatch(logoutUser(response))))
+
```
Comment (yunhsincynthiachen) [inline src/actions/actions.jsx]: I get lint errors here

**Review by yunhsincynthiachen — CHANGES_REQUESTED** [review]
Check out your lint errors. But everything else looks good!

File: routes/index.js [hunk 3]
```diff
@@ -32,6 +32,13 @@ router.post('/login', passport.authenticate('local'),
     });
   });
 
+// logout anyone who is logged in
+router.get('/logout',
+  (req, res) => {
+    req.logout();
+    res.send(200);
```
Comment (yunhsincynthiachen) [inline routes/index.js]: Use res.sendStatus(status) instead

**Comment by yunhsincynthiachen** [comment]
get '/requirements' and '/completedcourses' still get 500 errors when logging out. i think it shouldn't call those to routes when re-rendering the whole login page, but this is the last thing. everything else looks good!!!

**Review by arianaolson419 — APPROVED** [review]

**Review by yunhsincynthiachen — APPROVED** [review]

**Pull request merged**